projects/fal/src/fal/sdk.py

```diff
@@ -39,8 +39,9 @@ def to_grpc(self) -> grpc.ChannelCredentials:
         raise NotImplementedError
 
     @property
-    def extra_options(self) -> list[tuple[str, str]]:
-        return GRPC_OPTIONS
+    def base_options(self) -> dict[str, str]:
+        grpc_ops: list[tuple[str, str]] = GRPC_OPTIONS
```

Comment:
I'd rather put the config here @chamini2 (inside `GRPC_OPTIONS`, so we dont have the parse and merge etc)